README.rst

```diff
@@ -36,18 +44,19 @@ System Engineering related document by IPMU team.
 16. SCR cabling design
```

Comment:
There seems a document about this topic:
https://github.com/Subaru-PFS/ics_doc/blob/master/SE-IPMU-00016.pptx

Reply:
good catch! added.